eap74-rhel8-payg/src/main/scripts/jbosseap-setup-redhat.sh

```diff
@@ -72,7 +75,7 @@ sudo yum install curl wget unzip vim git -y | log; flag=${PIPESTATUS[0]}
 if [[ "${JDK_VERSION,,}" == "openjdk17" ]]; then
     echo "sudo alternatives --set java java-17-openjdk.x86_64" | log; flag=${PIPESTATUS[0]}
     sudo alternatives --set java java-17-openjdk.x86_64| log; flag=${PIPESTATUS[0]}
-elif [[ "${JDK_VERSION,,}" == "openjdk11" ]]; then
+elif t[[ "${JDK_VERSION,,}" == "openjdk11" ]]; then
```

Comment:
Hello @hcherukuri , I think we should remove the `t`

Reply:
Yes, great catch @backwind1233 .

Reply:
Fixed in 64a0021.